examples/azure-container-apps/Pulumi.yaml

```diff
@@ -14,13 +14,11 @@ variables:
           resourceGroupName: ${resourceGroup.name}
           workspaceName: ${workspace.name}
         Return: primarySharedKey
-  adminUsername:
-    Fn::Invoke:
-      Function: azure-native:containerregistry:listRegistryCredentials
-      Arguments:
-        resourceGroupName: ${resourceGroup.name}
-        registryName: ${registry.name}
-      Return: username
+  adminRegistryCreds:
+    Fn::azure-native:containerregistry:listRegistryCredentials:
+      resourceGroupName: ${resourceGroup.name}
+      registryName: ${registry.name}
+  adminUsername: ${adminRegistryCreds.username}
```

Comment:
Can we replace `adminPasswords` with `Fn::Secret: ${adminRegistryCreds.password}`?